gensim/models/hdpmodel.py

```diff
@@ -80,65 +122,125 @@ def lda_e_step(doc_word_ids, doc_word_counts, alpha, beta, max_iter=100):
+        Flag to indicate whether `lambda `and :math:`E[log \\theta]` have been updated if True, otherwise - not.
+    m_timestamp : numpy.ndarray
+        Helps to keep track and perform lazy updates on lambda.
+    m_updatect : int
+        Keeps track of current time and is incremented everytime :meth:`~gensim.models.hdpmodel.HdpModel.update_lambda`
+        is called.
+    m_var_sticks : numpy.ndarray
+        Array of values for stick.
+    m_varphi_ss : numpy.ndarray
+        Used to update top level sticks.
+    m_W : int
+        Length of dictionary for the input corpus.
 
     """
-
     def __init__(self, corpus, id2word, max_chunks=None, max_time=None,
                  chunksize=256, kappa=1.0, tau=64.0, K=15, T=150, alpha=1,
                  gamma=1, eta=0.01, scale=1.0, var_converge=0.0001,
                  outputdir=None, random_state=None):
         """
-        `gamma`: first level concentration
-        `alpha`: second level concentration
-        `eta`: the topic Dirichlet
-        `T`: top level truncation level
-        `K`: second level truncation level
-        `kappa`: learning rate
-        `tau`: slow down parameter
-        `max_time`: stop training after this many seconds
-        `max_chunks`: stop after having processed this many chunks (wrap around
-        corpus beginning in another corpus pass, if there are not enough chunks
-        in the corpus)
+        Parameters
+        ----------
+        corpus : iterable of list of (int, float)
+            Corpus in BoW format.
+        id2word : :class:`~gensim.corpora.dictionary.Dictionary`
+            Dictionary for the input corpus.
+        max_chunks : int, optional
+            Upper bound on how many chunks to process. It wraps around corpus beginning in another corpus pass,
+            if there are not enough chunks in the corpusю
+        max_time : int, optional
+            Upper bound on time (in seconds) for which model will be trained.
+        chunksize : int, optional
+            Number of documents in one chunck
+        kappa : float, optional
+            Learning rate
+        tau : float, optional
+            Slow down parameter
+        K : int, optional
+            Second level truncation level
+        T : int, optional
+            Top level truncation level
+        alpha : int, optional
+            Second level concentration
+        gamma : int, optional
+            First level concentration
+        eta : float, optional
+            The topic Dirichlet
+        scale : float, optional
+            Weights information from the mini-chunk of corpus to calculate rhot.
+        var_converge : float, optional
+            Lower bound on the right side of convergence. Used when updating variational parameters for a
+            single document.
+        outputdir : str, optional
+            Stores topic and options information in the specified directory.
+        random_state : :class:`~np.random.RandomState`, optional
```

Comment:
are you sure about type?

Reply:
Actually the parameter's type is `{None, int, array_like}` but the attribute type is the one I mentioned.I got it from [here](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/utils.py#L56). Should I go with the parameter's type ?

Reply:
you can mention all of this (mentioned 3 + current)